Bio/AlignIO/PhylipIO.py

```diff
@@ -441,3 +429,19 @@ def __next__(self):
                              id=i, name=i, description=i)
                    for (i, s) in zip(ids, seqs))
         return MultipleSeqAlignment(records, self.alphabet)
+
+
+def sanitize_name(name, width=None):
+    """Sanitise sequence identifier for output.
+
+    Removes the banned characters "[]()" and replaces the characters ":;"
+    with "|". The name is not to "width" characters if specified.
```

Comment:
Typo? The name is *truncated* to "width" characters if specified.

Reply:
Thanks -- fixed in 230d68f.